tools/ReleaseNotes/release_editor.py

```diff
@@ -0,0 +1,129 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2022 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import os
+import sys
+import shutil
+
+DIRECTIVE = ".. amalgamate:: "
+
+
+def getReleaseRoot():
+    script_dir = os.path.split(sys.argv[0])[0]
+    return os.path.abspath(os.path.join(script_dir, '../../docs/source/release/'))
+
+
+def fixReleaseName(name):
+    if not name.startswith('v'):
+        name = 'v' + name
+    return name
+
+
+def createFileLocation(releaseNo):
+    release_root = getReleaseRoot()
```

Comment:
If we make the `pathllib` modification then this can be simply:

```python
def createFileLocation(releaseNo) -> pathlib.Path:
    return getReleaseRoot() / releaseNo
```

`pathlib` means we can use `/` but Python handles using a `\` on Windows paths.